### PR TITLE
Hc 210

### PR DESCRIPTION
**Pull request by bnp26**
HC-210:
- updated notify_by_email.py to be python3 compatible.
- fixed "doc" access
- updated encoding
- updated job specs for all lightweight job calls
- fixed call to notify_by_email.py by notify_by_email.sh 
- Tested by running in tosca and figaro

### CODE REVIEW

**Review by pymonger — DISMISSED** [review]

**Comment by DustinKLo** [comment]
want to point out that the job may complete but the url will not work with the new UI
as pointed out in the ticket: https://hysds-core.atlassian.net/browse/HC-210

**Comment by pymonger** [comment]
@DustinKLo: good catch! 
@bnp26: let's tag up on what needs to be done to get that in sync with hysds_ui.

File: notify_by_email.py [hunk 1]
```diff
@@ -170,90 +182,99 @@ def get_facetview_link(link, _id, version=None):
...
         es = get_mozart_es()
         index = app.conf["STATUS_ALIAS"]
         facetview_url = app.conf["MOZART_URL"]
+        facetview_url = "/".join(facetview_url.split("/")[0:-1]) + "/figaro"
     else:  # "tosca"
         es = get_grq_es()
         index = app.conf["DATASET_ALIAS"]
-        facetview_url = "https://aria-search-beta.jpl.nasa.gov/search"  # TODO: why is it hard coded
+        facetview_url = app.conf["MOZART_URL"]
+        facetview_url = "/".join(facetview_url.split("/")[0:-1]) + "/tosca"
```
Comment (DustinKLo) [inline notify_by_email.py]: i think there should be a `/hysds_ui/` between the mozart url and `/figaro` (also `/tosca`)

**Review by DustinKLo — APPROVED** [review]
LGTM 👍

**Review by pymonger — APPROVED** [review]

**Pull request merged**